test/integration/editors/triggered/CommunicationMappingPlugin.test.ts

```diff
@@ -0,0 +1,192 @@
+        )
+      )).click();
+      await parent.updateComplete;
+      await new Promise(resolve => setTimeout(resolve, 100)); // await animation
+      expect(parent.wizardUI.dialogs.length).to.equal(0);
+    });
+
+    it('groups connections to unique pair of source ied, sink ied and control block', () => {
+      expect(
+        parent.wizardUI.dialog?.querySelectorAll('mwc-list-item').length
+      ).to.equal(4);
+    });
+
+    it('indicates the control block type with mwc-list graphic slot', () => {
+      expect(
+        parent.wizardUI.dialog!.querySelectorAll('mwc-list-item > mwc-icon')
+          .length
+      ).to.equal(4);
+    });
+
+    it('show the source ied, sink ied and control block', () => {
+      expect(
+        parent.wizardUI.dialog!.querySelectorAll('mwc-list-item > mwc-icon')
+          .length
+      ).to.equal(4);
+    });
+  });
+
+  describe('control block connection wizard', () => {
+    let commMappings: List;
+    beforeEach(async () => {
+      await element.trigger();
+      await parent.updateComplete;
+      await new Promise(resolve => setTimeout(resolve, 100)); // await animation
+      commMappings = <List>(
+        parent.wizardUI.dialog?.querySelector('filtered-list')
+      );
+    });
+
+    it('opens on Connection item click', async () => {
+      commMappings.items[0].click();
+      await parent.updateComplete;
+      expect(parent.wizardUI.dialogs.length).to.equal(1);
+      expect(parent.wizardUI.dialog?.heading).to.equal('IED2 - IED1');
+    });
+
+    it('shows all ClientLNs', async () => {
+      commMappings.items[0].click();
+      await parent.updateComplete;
+      expect(
+        parent.wizardUI.dialog?.querySelectorAll('mwc-check-list-item').length
+      ).to.equal(2);
+    });
+
+    it('removes selected ClientLNs', async () => {
+      expect(
+        doc.querySelector(
+          'ClientLN[iedName="IED1"][ldInst="CircuitBreaker_CB1"][lnClass="CSWI"]'
+        )
+      ).to.exist;
+      commMappings.items[0].click();
+      await parent.updateComplete;
+      await new Promise(resolve => setTimeout(resolve, 100)); // await animation
+      (<List>(
+        parent.wizardUI.dialog?.querySelector('filtered-list')
+      )).items[1].click();
+      await parent.updateComplete;
+      (<HTMLElement>(
+        parent.wizardUI.dialog?.querySelector(
+          'mwc-button[slot="primaryAction"]'
+        )
+      )).click();
+      await parent.updateComplete;
+      expect(
+        doc.querySelector(
+          'ClientLN[iedName="IED1"][ldInst="CircuitBreaker_CB1"][lnClass="CSWI"]'
+        )
+      ).to.be.null;
+    });
+
+    it('shows all ExtRefs', async () => {
+      commMappings.items[1].click();
+      await parent.updateComplete;
+      expect(
+        parent.wizardUI.dialog?.querySelectorAll('mwc-check-list-item').length
+      ).to.equal(4);
+    });
+
+    it('removes selected ExtRefs in case no intAddr is present', async () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
    it("removes selected ExtRefs in case no intAddr is present", async () => {
```